test/es2017/.eslintrc.json

```diff
@@ -1,5 +1,6 @@
 {
 		"parserOptions": {
-				"ecmaVersion": 2017
+				"ecmaVersion": 2018,
```

Comment:
worth keeping separate 2017 and 2018 test suites? or just renaming/rev'ing the folder name?

Reply:
Seems vaguely odd to allow ES2018 in a folder named es2017

Reply:
Agreed, I'll rename the directory. In a follow-up I'll try to reorganize the tests more generally since it's quite a mess at the moment.